curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/657e76356b6af6c07fe338c1.md

```diff
@@ -3,18 +3,14 @@ id: 657e76356b6af6c07fe338c1
 title: Task 28
 challengeType: 22
 dashedName: task-28
-audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
 ---
 
-<!--
-AUDIO REFERENCE: 
-Linda: I've been a cyclist for more than 15 years. It's a love that hasn’t disappeared with time.
--->
+<!-- (Audio) Linda: I've been a cyclist for more than 15 years. It's a love that hasn’t disappeared with time. -->
 
 # --description--
 
-In the Present Perfect Continuous Tense, you can use `for` and `since` to talk about time. 
-`For` tells how long something has been happening. `Since` tells when something started. 
+In the Present Perfect Tense, you can use `for` and `since` to talk about time. 
```

Comment:
```suggestion
In the Present Perfect Tense, `for` and `since` are used to discuss time duration and starting points.
`For` indicates the length of time an action has been ongoing. `Since` specifies the moment when the action began.
```